app/src/main/java/org/xbmc/kore/ui/sections/favourites/FavouritesListFragment.java

```diff
@@ -0,0 +1,224 @@
+package org.xbmc.kore.ui.sections.favourites;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.annotation.LayoutRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.GridView;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import org.xbmc.kore.R;
+import org.xbmc.kore.host.HostManager;
+import org.xbmc.kore.jsonrpc.ApiCallback;
+import org.xbmc.kore.jsonrpc.ApiList;
+import org.xbmc.kore.jsonrpc.method.Favourites;
+import org.xbmc.kore.jsonrpc.method.GUI;
+import org.xbmc.kore.jsonrpc.method.Player;
+import org.xbmc.kore.jsonrpc.type.FavouriteType;
+import org.xbmc.kore.utils.LogUtils;
+import org.xbmc.kore.utils.UIUtils;
+
+import java.util.List;
+
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+
+public class FavouritesListFragment extends Fragment implements SwipeRefreshLayout.OnRefreshListener {
```

Comment:
Well, I was taking the PVRChannelsListFragment as a reference, and it was extending just Fragment, so I made it similar to PVRChannelsListFragment. Would you like me to extend from AbstractListFragment and do the necessary changes?